Contributing.md

```diff
@@ -82,4 +82,23 @@ to the TC and the TC uses its standard consensus seeking process to evaluate whe
 not to add this new member. Members who do not participate consistently at the level of
 a majority of the other members are expected to resign.
 
-
+# Project Captains
+
+The Express TC can designate captains for individual projects/repos in the
+three organizations. These captains are responsible for being the primary
```

Comment:
In case we ever add more orgs, let’s remove the “three”.

Reply:
@wesleytodd - done, removed the word